ent/schema/statement.go

```diff
@@ -38,6 +38,7 @@ func (Statement) Fields() []ent.Field {
 func (Statement) Edges() []ent.Edge {
 	return []ent.Edge{
 		edge.To("subjects", Subject.Type).Annotations(entgql.RelayConnection()),
+		edge.To("policies", AttestationPolicy.Type).Annotations(entgql.RelayConnection()),
```

Comment:
This is a question for someone who knows more about `ent` than I do but, if policies -> statements are a 1:1 mapping, do we need the `RelayConnection()`? Maybe related, should we call the edge the singular `policy`?

Reply:
Yes, it makes sense.
I will wait for @mikhailswift, so if I need to make more changes :)

Reply:
The general way I think about whether an edge should be a RelayConnection is whether you want to query that edge directly or use it as an entry to the graph. We don't do so for the `attestation_collections` edge below, so I'd be inclined to keep it similar. Similarly to that, we probably want the `policies` edge to be `Unique`, since they are 1:1.

Reply:
Thanks!
I have added the new commit https://github.com/in-toto/archivista/pull/234/commits/04e6d303a858b6bda6c76be991fcce6be831eb57